assets/templates/list-item.html

```diff
@@ -1,6 +1,6 @@
 <li class="document-snippet">
 	<div class="badge-icon {{documentType}}"></div>
-  <a href="{{actionUrl}}" target="_blank" class="document-snippet-link" data-ga-document="" data-ga-document-type="{{documentType}}" data-ga-provider="{{providerName}}">
+  <a href="{{actionUrl}}" target="_blank" class="document-snippet-link" data-mixpanel-document-type="{{documentType}}" data-mixpanel-provider="{{providerName}}">
```

Comment:
If we are moving ga-helper to analytics-helper, should we rename data-ga to data-analytics or something like that ?